iron-overlay-behavior.html

```diff
@@ -391,8 +396,9 @@
     _finishPositioning: function() {
       this.style.display = 'none';
       this.style.transform = this.style.webkitTransform = '';
-      // force layout to avoid application of transform
-      /** @suppress {suspiciousCode} */ this.offsetWidth;
+      // Force layout layout to avoid application of transform.
+      // Set offsetWidth to itself so that compilers won't remove it.
+      this.offsetWidth = this.offsetWidth;
```

Comment:
Ok just checked with Levi and it's fine (you only do a style recalc if you need to, and after the first `offsetWidth` the second will be the same). But are you sure we shouldn't leave `@suppress {suspiciousCode}` in, in case the compiler still wants to remove it as a no-op?